app/assets/javascripts/discourse/app/templates/modal/sidebar-section-form.hbs

```diff
@@ -26,6 +26,7 @@
             @value={{link.icon}}
             @options={{hash maximum=1}}
             class={{link.iconCssClass}}
+            @available={{true}}
```

Comment:
I'd rename it to `onlyAvailable`. w/o context, `@available` looks like you want the IconPicker to "be available" whatever that would mean 😅 `onlyAvailable` (or `showOnlyAvailable` or `onlyAvailableIcons`) provides a bit more of a hint about this option 😃